tests/python/frontend/pytorch/test_forward.py

```diff
@@ -3437,6 +3437,12 @@ def test_fn(x, weights=None):
     verify_trace_model(test_fn, [inp, weights], targets)
 
 
+def test_hard_swish():
+    examples = [torch.rand(8).float(), torch.rand(8, 10).float(), torch.rand(1, 1, 10).float()]
+    for input in examples:
+        verify_model(torch.nn.Hardswish().eval(), input_data=input)
```

Comment:
Any reason to drop `hard_swish_inplace`?

Reply:
hard_swish_inplace is not a pure function. It will change  input tensor

Reply:
I can modify verify_model function

Reply:
Looks great!